languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -35467,6 +35467,26 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TOURS_CRITIQUES" name="Tours critiqués divers (barbarismes, impropriétés, solécismes, etc.)">
+      <rule id="QU_EST_CE_QUE" name="qu’est-ce que" default="temp_off">
+          <antipattern>
+              <token regexp="yes">qu(&apostrophe;)</token>
+              <token>est</token>
+              <token>-ce</token>
+              <token regexp="yes" skip="-1">qu.*</token>
+              <token postag="M fin inte"/>
+          </antipattern>
+          <pattern>
+              <marker>
+                  <token regexp="yes">qu(&apostrophe;)</token>
+                  <token>est</token>
+                  <token>-ce</token>
+                  <token regexp="yes">qu(&apostrophe;)|que</token>
+              </marker>
```

Comment:
The spacing problem seems to be resolved (or there never was one). But the unnecessary markers are still there.

Reply:
I made the necessary changes so the rule is done.